ci/scripts/byfn_custom.sh

```diff
@@ -0,0 +1,10 @@
+#!/bin/bash
+# Copyright the Hyperledger Fabric contributors. All rights reserved.
+#
+# SPDX-License-Identifier: Apache-2.0
+set -euo pipefail
```

Comment:
Consider removing the "-e" on the byfn_ and trigger_ scripts. If the test fails during or after launching the network, then you probably want to execute "down" to cleanup anyways, and to collect whatever artifacts you can even if they are not all there.